packages/tasks/src/model-libraries.ts

```diff
@@ -491,6 +491,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"pt"`,
 	},
+	medvae: {
+		prettyLabel: "MedVAE",
+		repoName: "MedVAE",
+		repoUrl: "https://github.com/StanfordMIMI/MedVAE",
+		filter: false,
+		countDownloads: `path_extension:"yaml" OR path_extension:"ckpt"`,
```

Comment:
That rule would count a download on each ckpt or yaml file download. I suppose that to run a model one has to download both files, correct? In that case, I'd recommend to count only on ckpt or only on yaml files to prevent double counting. WDYT?

```suggestion
		countDownloads: `path_extension:"ckpt"`,
```